ui/segmented-bar.reel/segmented-bar.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * @module montage/ui/segmented-bar.reel
+ * @requires montage/ui/base/abstract-button.AbstractButton
+ * @requires montage/core/tree-controller.TreeController
+ */
+var AbstractButton = require("ui/base/abstract-button").AbstractButton;
+
+/**
+ * @class SegmentedBar
+ * @classdesc
+ * A segmented bar.
+ * Reference design: OS X storage space visualization found in iTunes or About This Mac
+ * Inheriting from AbstractButton because SegmentedBar needs its PressComposer behavior
+ *
+ * @extends AbstractButton
+ *
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Flexible_boxes}
+ * for FlexBox usage
+ */
+exports.SegmentedBar = AbstractButton.specialize( /** @lends SegmentedBar.prototype # */ {
```

Comment:
> OK, now I understand. I didn't know the part about
repetition.cancelBindings(), as it hadn't occurred to me to call that.

> These content/contentController bindings that are broken here, when will
they be restored? Does templateDidLoad() get called again when a component
is reused after an exitDocument? I suspect not.

> This dynamic destruction of bindings is a code smell... why is it here?
We're trying to save the memory space for one binding when a SegmentedBar
is temporarily taken out of a document? Is that significant?

> It seems better to just have the repetition.contentController binding in
the serialization and not mess with it.

Will fix per discussion